types/react-navigation/index.d.ts

```diff
@@ -633,6 +634,19 @@ export class Transitioner extends React.Component<
   TransitionerState
 > { }
 
+
+/**
+ * Tab Router
+ *
+ * @desc from react-navigation/src/routers/TabRouter.js
+ * @param routeConfigs
+ * @param config
+ */
+export function TabRouter(
+  routeConfigs: NavigationRouteConfigMap,
+  config: NavigationTabRouterConfig
+): NavigationRouter<any, any, any>
+
 /**
```

Comment:
`StackRouter` is familar with `TabRouter`.
If it's convenient, would you add StackRouter too.
😄